packages/rum-core/src/rawRumEvent.types.ts

```diff
@@ -101,6 +101,7 @@ export interface RawRumViewEvent {
   }
   _dd: {
     document_version: number
+    replay?: ViewReplayStats
```

Comment:
it seems that we have several namings representing the same thing:

- replay
- viewStats
- viewReplayStats

could we use the same naming everywhere?

Reply:
what about `replayStats` everywhere?

Reply:
Ok!